src/main/kotlin/com/exactpro/th2/service/generator/protoc/java/ServiceDefaultImplGenerator.kt

```diff
@@ -58,13 +58,13 @@ class ServiceDefaultImplGenerator : AbstractJavaServiceGenerator(), Generator {
 
     private fun generateBlockingDefaultImpl(service: ServiceDescriptorProto, javaPackage: String, messageNameToJavaPackage: Map<String, String>): FileSpec {
         val javaFile = service.methodList.map {
-            MethodSpec.methodBuilder(it.name)
+            MethodSpec.methodBuilder(it.name.decapitalize())
```

Comment:
Extract it to a separate variable instead of duplicate call of `decapitalize` method